packages/text-input/src/TextInput.tsx

```diff
@@ -258,16 +261,17 @@ function getStatefulInputStyles(state: State, optional: boolean) {
 const TextInput = React.forwardRef(
   (
     {
-      id: propsId,
       label,
       description,
-      optional = false,
-      disabled = false,
       onChange,
       placeholder,
       errorMessage,
+      optional = false,
```

Comment:
Reorg'd the props, bc I like to have to aliased props grouped together as well as the props that we provide a default value for grouped together